composer.lock

```diff
@@ -246,6 +246,60 @@
     ],
     "packages-dev": [
         {
+            "name": "doctrine/instantiator",
```

Comment:
Looks like they can be traced to phpunit or phpspec/prophecy (which is included by phpunit).

```sh
# new packages
$ git show 30e9cd515c9e801d4ae93b133a590226da16c55d composer.lock  | grep '+.*name.*/' | cut -d \" -f 4
doctrine/instantiator
phpdocumentor/reflection-common
phpdocumentor/reflection-docblock
phpdocumentor/type-resolver
phpspec/prophecy
sebastian/comparator
sebastian/diff
sebastian/environment
sebastian/exporter
sebastian/global-state
sebastian/recursion-context
sebastian/version
webmozart/assert

NEWPACKAGES=`git show 30e9cd515c9e801d4ae93b133a590226da16c55d composer.lock  | grep '+.*name.*/' | cut -d \" -f 4`

# Run 'why' for each package
for package in $NEWPACKAGES; do echo "******checking $package********"; composer why $package; done
******checking doctrine/instantiator********
phpspec/prophecy              v1.6.1  requires  doctrine/instantiator (^1.0.2)  
phpunit/phpunit-mock-objects  2.3.8   requires  doctrine/instantiator (^1.0.2)  
******checking phpdocumentor/reflection-common********
phpdocumentor/reflection-docblock  3.1.0  requires  phpdocumentor/reflection-common (^1.0@dev)  
phpdocumentor/type-resolver        0.2    requires  phpdocumentor/reflection-common (^1.0)      
******checking phpdocumentor/reflection-docblock********
phpspec/prophecy  v1.6.1  requires  phpdocumentor/reflection-docblock (^2.0|^3.0.2)  
******checking phpdocumentor/type-resolver********
phpdocumentor/reflection-docblock  3.1.0  requires  phpdocumentor/type-resolver (^0.2.0)  
******checking phpspec/prophecy********
phpunit/phpunit  4.8.27  requires  phpspec/prophecy (^1.3.1)  
******checking sebastian/comparator********
phpspec/prophecy  v1.6.1  requires  sebastian/comparator (^1.1)  
phpunit/phpunit   4.8.27  requires  sebastian/comparator (~1.1)  
******checking sebastian/diff********
phpunit/phpunit       4.8.27  requires  sebastian/diff (~1.2)  
sebastian/comparator  1.2.0   requires  sebastian/diff (~1.2)  
******checking sebastian/environment********
phpunit/php-code-coverage  2.2.4   requires  sebastian/environment (^1.3.2)  
phpunit/phpunit            4.8.27  requires  sebastian/environment (~1.3)    
******checking sebastian/exporter********
phpunit/phpunit               4.8.27  requires  sebastian/exporter (~1.2)  
phpunit/phpunit-mock-objects  2.3.8   requires  sebastian/exporter (~1.2)  
sebastian/comparator          1.2.0   requires  sebastian/exporter (~1.2)  
******checking sebastian/global-state********
phpunit/phpunit  4.8.27  requires  sebastian/global-state (~1.0)  
******checking sebastian/recursion-context********
phpspec/prophecy    v1.6.1  requires  sebastian/recursion-context (^1.0)  
sebastian/exporter  1.2.2   requires  sebastian/recursion-context (~1.0)  
******checking sebastian/version********
phpunit/php-code-coverage  2.2.4   requires  sebastian/version (~1.0)  
phpunit/phpunit            4.8.27  requires  sebastian/version (~1.0)  
******checking webmozart/assert********
phpdocumentor/reflection-docblock  3.1.0  requires  webmozart/assert (^1.0) 
```